packages/pilot/src/containers/AddRecipient/ErrorStep/ErrorButtons.js

```diff
@@ -28,6 +30,21 @@ const ErrorButtons = ({
     )
   }
 
+  if (error === CANNOT_CREATE_RECIPIENT_ERROR) {
+    return (
+      <>
+        <div className={style.errorDetails}>
+          <p>{t('phone')}: <a href="tel:3004-9709">3004-9709</a></p>
```

Comment:
Aqui é bom usar o [formato internacional](https://developers.google.com/web/fundamentals/native-hardware/click-to-call#use_the_international_dialing_format)

```suggestion
          <p>{t('phone')}: <a href="tel:+55-<ddd>-3004-9709">3004-9709</a></p>
```

Reply:
É q esse é um telefone sem DDD mesmo Vaguinho. Eu nao sei como funciona, mas alguns telefones na stone nao tem DDD, valem pro Brasil inteiro.